controllers/articles.controller.js

```diff
@@ -31,3 +32,17 @@ exports.getArticleComments = (req, res, next) => {
 
     .catch(next);
 };
+
+exports.postComment = (req, res, next) => {
+  const { article_id } = req.params;
+  const { username, body } = req.body;
+
+  if (!username || !body) {
+    return res.status(400).send({ msg: "Bad request" });
+  }
```

Comment:
You don't need to manually check these, if you attempted the query with these missing psql would throw an error itself!